### DISCUSSION TITLE
How to jump to certain line of dialogue?

### DISCUSSION THREAD

**Discussion by kevinsavala** [nathanhoad/godot_dialogue_manager#427] [Q&A] [answered]
I'm currently making a visual novel but I can only start the dialogue balloon from tags like ~start and ~end. I'm trying to save a line number and then jump to that line when the balloon starts. Maybe it's possible to create tags dinamically? That could help. Thanks.

**Comment by nathanhoad** [comment, marked as answer]
You can jump to any line's ID to start from there (titles are just a more convenient way of referring to meaningful start points). You can see this in action [in the example balloon](https://github.com/nathanhoad/godot_dialogue_manager/blob/main/addons/dialogue_manager/example_balloon/example_balloon.gd#L82); both `start` and `next` use the same function to request the next dialogue line, one using a title and the other an ID.
